lib/msal-angular/package.json

```diff
@@ -75,4 +75,4 @@
     "@azure/msal-browser": "^3.0.0-alpha.1",
     "rxjs": "^7.0.0"
```

Comment:
Was this an issue specific to our sample or do we need to update the peer dependency as well?

Reply:
This was specifically for our CI E2E tests and shouldn't affect our customers. Samples with different RxJS versions work fine with the peer dependency of ^7.0.0.